signing_service/signing_service/utils.py

```diff
@@ -112,5 +115,41 @@ def __call__(  # type: ignore
 
 class Notifier(object):
 
-    def send(self) -> None:
-        pass
+    def __init__(
+        self,
+        from_email_address: str,
+        from_email_password: str,
+        to_email_addresses: list,
+    ) -> None:
+        self.from_email_address = from_email_address
+        self.from_email_password = from_email_password
+        self.to_email_addresses = to_email_addresses
+
+    def send(
+        self,
+        message: str,
+    ) -> None:
+        message_to_send = MIMEMultipart()
+        message_to_send['From'] = "Signing Service Notifier"
+        message_to_send['To'] = ','.join(self.to_email_addresses)
+        message_to_send['Subject'] = "Signing Service notification"
+        message_to_send.attach(MIMEText(message))
+        if self.from_email_address is not None and self.from_email_password is not None:
+            self.send_email(message_to_send)
+        print(message_to_send)
+        del message_to_send
+
+    def send_email(
+        self,
+        message: str,
+    ) -> None:
+        try:
+            server = smtplib.SMTP_SSL('smtp.gmail.com:465', timeout=3)
```

Comment:
How about starting server and login in `__init__()`, so it will be done once?
Same for `quit()`, I think it would be best to make a separate method for it and connect it to `SigningService` shutdown handlers (shutdown signal, exception handler etc.) so it will go down along with `SigningService`.

Reply:
I was concern that somehow we can lose connection to gmail `smtp`, when Signing Service is still up and running. But okey it's definitely possible to split it to separate method.

Reply:
Now that you have mentioned it, if that happen and server is not able to reconnect by itself, it might be a problem. I am not sure if we should change it then. After all this email will be sent at most 2 times a day, right? 
@dybi what do you think?

Reply:
Ok. I've separated `connect_to_gmail_smtp_server()` and `end_smtp_server_connection()` function from `send_email()`, but adding `self.server` to the class "upset" `mypy` a little bit, so take another look at this.